cvmfs/catalog_mgr_rw_impl.h

```diff
@@ -0,0 +1,181 @@
+/**
+ * This file is part of the CernVM file system.
+ */
+
+
+#ifndef CVMFS_CATALOG_MGR_RW_IMPL_H_
+#define CVMFS_CATALOG_MGR_RW_IMPL_H_
+
+#include "catalog_mgr_rw.h"
+
+#include <inttypes.h>
+#include <unistd.h>
+
+#include <cassert>
+#include <cstdio>
+#include <cstdlib>
+#include <ctime>
+#include <string>
+#include <vector>
+
+#include "catalog_rw.h"
```

Comment:
It looks to me as there are many includes in this file which are not really necessary. Pleas re-check.